cmd/uninstall.go

```diff
@@ -24,7 +24,7 @@ var uninstallCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(uninstallCmd)
 
-	uninstallCmd.Flags().StringVarP(&uninstallOptions.Namespace, "namespace", "n", "kubearmor", "Namespace for resources")
+	uninstallCmd.Flags().StringVarP(&uninstallOptions.Namespace, "namespace", "n", "", "Namespace for resources")
```

Comment:
I think we should add to the help message what would the default behavior be when no namespace is specified? :thinking: